blackjax/inference/metrics.py

```diff
@@ -88,23 +80,19 @@ def kinetic_energy(momentum: PyTree, *_) -> float:
         mass_matrix_sqrt = jscipy.linalg.solve_triangular(
             tril_inv, identity, lower=True
         )
-
-        def momentum_generator(rng_key: jax.random.PRNGKey, position: PyTree) -> PyTree:
-            _, unravel_fn = ravel_pytree(position)
-            std = jax.random.normal(rng_key, shape)
-            momentum = jnp.dot(std, mass_matrix_sqrt)
-            return unravel_fn(momentum)
-
-        def kinetic_energy(momentum: PyTree, *_) -> float:
-            momentum, _ = ravel_pytree(momentum)
-            momentum = jnp.array(momentum)
-            velocity = jnp.matmul(inverse_mass_matrix, momentum)
-            return 0.5 * jnp.dot(velocity, momentum)
-
-        return momentum_generator, kinetic_energy
-
-    else:
-        raise ValueError(
-            "The mass matrix has the wrong number of dimensions:"
-            f" expected 1 or 2, got {jnp.ndim(inverse_mass_matrix)}."
-        )
+        dot, matmul = jnp.dot, jnp.matmul
+
+    def momentum_generator(rng_key: jax.random.PRNGKey, position: PyTree) -> PyTree:
+        _, unravel_fn = ravel_pytree(position)
+        std = jax.random.normal(rng_key, shape)
```

Comment:
What does std stand for? Standard deviation?

Reply:
Yeah. It's a terrible name when you think too much about it.

Reply:
Maybe it should be named something like random sample, or momentum_sample?

Reply:
Oh no it stands for standard normal I believe.

Reply:
Would you be opposed if it was renamed standard_normal to make it unambiguous?